packages/@expo/cli/src/install/checkPackages.ts

```diff
@@ -73,13 +73,17 @@ export async function checkPackagesAsync(
    *       and you cannot replace this guard with a try/catch around the import('expo-router')
```

Comment:
oh i see, i missed that

Reply:
nm i think that's fine. i think it is just saying that you can't use try/catch for the import because we'll always have expo-router available in our repo - so we need to check if it's in dependencies